tests/framework/installer/ceph_manifests.go

```diff
@@ -30,6 +30,8 @@ const (
 	Version1_2 = "v1.2.7"
```

Comment:
Shouldn't this be removed as well, or is this constant used somewhere else?

Reply:
Yeah. Caught that as soon as I created the PR. Done.